### ISSUE TITLE
A11y_CombineStatusandDataUsagesettingspages_Status_NetworkStatus_ScreenReader: The word "Heading" is not announced for the respective headings.

### ISSUE THREAD

**Issue by fsteffi** [nvaccess/nvda#10207]
**Environment Details:​​**
Application name: Combine Status and Data Usage settings pages
Screen reader: Nvda2019.2

**Repro Steps:​​**
1. Go to "Windows search bar" and search for "settings".
2. Click on "Network and internet"​
3. Navigate to "status tab"​
4. Check for network status​
5. Launch screen reader as nvda
6. Navigate using insert+4 to the heading " Network status"

**Actual:​**
On navigating on the page using insert+4 to the header "Network status", nvda announces it as "Network status" the word "header" is not announced along with it. 
This issue is not observed in narrator.

**Same issue observed on the following page:**
1.Advanced Network settings
2.Data Usage

**Expected:​**
On navigating on the page using insert+4 the heading " Network status" should be announced as heading along with its name and heading level.

**User Impact:​**
Screen reader users will not know that the data announced is a header which contains data, the data that will be announced there after is associated to that header.

**Attachment Reference:**
![23438684_A11y_CombineStatusandDataUsagesettingspages_Status_NetworkStatus_ScreenReader](https://user-images.githubusercontent.com/24580570/64761718-5fa60700-d55a-11e9-99e8-365151d92550.png)

**Comment by Adriani90** [comment]
This works as expected only if the add-on windows 10 apps is installed in NVDA.
@josephsl are you planning to donate the coresponding code for this issue via a PR to the NVDA core? I guess this is not a very big amount of code. Isn't it?

**Comment by Adriani90** [comment]
@josephsl what's your view on my last comment above?

**Comment by Adriani90** [comment]
@josephsl do you have any plans to add this to the core as asked in https://github.com/nvaccess/nvda/issues/10207#issuecomment-633722673?

**Comment by josephsl** [comment]
Hi,

It makes a bit more sense to add support for XAML heading roles if keyboard focus is possible, but since it is not (that's what is shown in the first comment), I'm a bit hesitant to add it. Besides, marking these controls as XAML headings (that's what they really are) suppresses other control role information such as a list item that is a heading and is a breadcrumb bar item. Thus, for now, I plan not to not bring this into NVDA - the relevant code was removed from Windows App Essentials add-on a while ago.

Thanks.
